Epsilon.Abstractions/Component/CompetenceProfile.cs

```diff
@@ -12,17 +13,216 @@ IEnumerable<ProfessionalSkillResult> ProfessionalSkillOutcomes
 {
     public void AddToWordDocument(MainDocumentPart mainDocumentPart)
     {
-        // TODO: This is simply an example to show the capability of the component architecture
         var body = new Body();
 
-        body.AppendChild(
-            new Paragraph(
-                new Run(
-                    new Text("Competence profile comes here")
-                    )
-                )
-            );
+        body.Append(new Paragraph(new Run(new Text(""))));
+        body.AppendChild(GetTasks());
+        body.Append(new Paragraph(new Run(new Text(""))));
+        body.AppendChild(GetSkills());
+        body.Append(new Paragraph(new Run(new Text(""))));
+        body.AppendChild(GetLegend());
+        body.Append(new Paragraph(new Run(new Text(""))));
```

Comment:
Are these `body.Append(new Paragraph(new Run(new Text(""))));` nessecary. It seems kind of bloated.

Reply:
If these are not included, all the tables will merge together, will be automatically fixed with the refactor

Reply:
Can you atleast replace it with something like a line break?